source/administration/plugins.rst

```diff
@@ -12,12 +12,63 @@ Consider using a plugin in the following scenarios:
+About Plugins
+------
 
-Browse available plugins at `https://integrations.mattermost.com/ <https://integrations.mattermost.com/>`__.
+Plugins may have one or both of the following parts:
+
+ - **Webapp plugins**: Customize the Mattermost user interface by adding buttons to the channel header, overriding the ``RHS``, or even rendering a custom post type within the center channel. All this is possible without having to fork the source code and rebase on every Mattermost release. For a sample plugin, see `our Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__.
+ - **Server plugins**: Run a Go process alongside the server, filtering messages or integrating with third-party systems such as Jira, GitLab or Jenkins. For a sample plugin, see `our Jira plugin <https://github.com/mattermost/mattermost-plugin-jira>`__.
+
+Plugin Marketplace
+-------
+The Plugin Marketplace is a collection of plugins that can greatly increase the value and capabilities of your Mattermost deployment.  End-users can see increased productivity through quick access to systems such as Jira, Zoom, Webex, and GitHub.  Mattermost Administrators can discover new plugins and quickly deploy them to their servers, including High availability clusters.  It is available in 5.16 and is accessed via **Main Menu > Plugin Marketplace**.  Our Marketplace listings will continue to expand as we add new features and plugins.  
+
+.. image:: https://user-images.githubusercontent.com/915956/66891467-1b18eb80-ef9e-11e9-9de3-37a3c5899bd8.png
+
+Installing a Plugin
+~~~~~~~~~~~~~~~~
+
+When a new plugin becomes available on the marketplace, a new listing with a button labeled **Install** will be displayed.  After clicking **Install**, the latest plugin binary is downloaded from its respective source and installed on the server.    
+
+Configuring and Enabling a Plugin
+~~~~~~~~~~~~~~~
+
+Once a plugin has been installed (or pre-installed if it shipped with Mattermost binary release) there will be a "Configure" button shown next to the plugin listing.  
+
+1. Click on the button and you will be navigated to the plugins "Settings" page.  
+2. Fill in Plugin settings as required.
+3. At the top of the page, set **Enable Plugin** setting to ``True``.  If this flag is not switched on, the plugin will never become active.
+4. Test out the plugin as needed.
+
+Upgrading Plugins
+~~~~~~~~~~~~~~~~~
+
+In v5.16, the Marketplace only supports the installation of new plugins.  To upgrade a plugin, you will need to manually update it by downloading the binary file from the GitHub repository and uploading it in the **System Console > Plugin management** section.  In v5.18 (Dec 2019), the Marketplace will allow you to upgrade a plugin on demand when a new version becomes available.
```

Comment:
nit: Should we be committing to timelines in documentation like this?
```suggestion
In v5.16, the Marketplace only supports the installation of new plugins.  To upgrade a plugin, you will need to manually update it by downloading the binary file from the GitHub repository and uploading it in the **System Console > Plugin management** section.  In an upcoming release, the Marketplace will allow you to upgrade a plugin on demand when a new version becomes available.
```

Reply:
I was on the fence, removing.